hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarbonitingMetadataFreezerActor.scala

```diff
@@ -50,10 +51,17 @@ class CarbonitingMetadataFreezerActor(freezingConfig: ActiveMetadataFreezingConf
 
   when(Fetching) {
     case Event(SuccessfulMetadataJsonResponse(_, responseJson), FetchingData(workflowId)) =>
-      asyncIo.writeAsync(carboniterConfig.makePath(workflowId), responseJson.prettyPrint, Seq(StandardOpenOption.CREATE), compressPayload = true) onComplete {
-        result => self ! CarbonitingFreezeResult(result)
+      val jsonAsString = responseJson.prettyPrint
+      val jsonStringBytesSize = jsonAsString.getBytes(StandardCharsets.UTF_8).length
+      if (jsonStringBytesSize <= carboniterConfig.bucketReadLimit) {
+        asyncIo.writeAsync(carboniterConfig.makePath(workflowId), jsonAsString, Seq(StandardOpenOption.CREATE), compressPayload = true) onComplete {
+          result => self ! CarbonitingFreezeResult(result)
+        }
+        goto(Freezing) using FreezingData(workflowId)
+      } else {
+        log.warning(s"Carbonited metadata length is $jsonStringBytesSize bytes, which is greater than configured bucket-read-limit-bytes value. Marking as $TooLargeToArchive")
```

Comment:
ToL: Warnings are sent off to Sentry. In this case though I will be curious to see how frequently we cannot archive the metadata.

If we need to later NOT send these messages to Sentry, we'll have to do one of:
- (Temporarily?) Mute the alert in Sentry
- Adjust the logback config
- Recompile the code to change this from warning to info